posix.mak

```diff
@@ -5,7 +5,7 @@ ECTAGS_FILES = src/*.[chd] src/backend/*.[chd] src/root/*.[chd] src/tk/*.[chd]
 .PHONY: all clean test install auto-tester-build auto-tester-test toolchain-info
 
 all:
-	$(QUIET)$(MAKE) -C src -f posix.mak
+	$(QUIET)$(MAKE) -C src -f posix.mak all
```

Comment:
This makes the default Makefile target explicit (`all` is the default target of `src/posix.mak`).

Reply:
Is there a disadvantage to assume the default target builds all? After all, if it's not it means we wrote that makefile poorly.

Reply:
Explicitly for the win - it was helpful during debugging this.